docker-compose.yml

```diff
@@ -39,6 +39,10 @@ services:
     image: maven:3-jdk-8
     volumes:
       - ./tmp/serverless-integration-test-aws-kotlin-jvm-maven:/app
+  aws-kotlin-node:
+    image: pgoudreau/docker-maven-node
```

Comment:
Out of curiosity: Is there a Docker image which includes `gradle` and `node` available? I was confused that this one is focused on `maven` and `node` 🤔

(but IMHO it's not important to change that)

Reply:
Docker image doesn't need global gradle installation because template ships with gradlew - gradle wrapper. My point here was to have a image with both JDK and Node. I'll change that to something more general.

Reply:
Alright. That makes sense. Thanks for the clarification :+1: